frontend-client/app/layout.tsx

```diff
@@ -18,9 +19,10 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body className={manrope.className}>
-        <div className="w-screen overflow-hidden">
+        <div className="flex h-screen w-screen flex-col overflow-hidden">
```

Comment:
지금 상태로라면 viewport의 height이 작다면 notice page 짤리고 header등도 이상하게 보일겁니다! 따라서 height을 100vh로 강제하지말고 최소 100vh에서 시작해서 그 이상으로도 늘어날 수 있도록 하고 overflow-y-auto 옵션을 줘서 세로로는 스크롤이 가능하게 하면 좋을 것 같아요!

```suggestion
        <div className="flex min-h-screen w-screen flex-col overflow-y-auto overflow-x-hidden">
```

Reply:
vh를 쓰면 모바일에서 스크롤이 생기는 문제가 있어서 저는 다른 옵션 쓰는 게 좋은 거 같아요. 여러 가지 있는데, 최근 CSS 신기능인 dvh 쓰는 게 좋아보입니다. [caniuse.com](https://caniuse.com/?search=dvh)에서도 지원 브라우저가 85% 정도라고 하니까 괜찮을 거 같아요.

https://s0ojin.tistory.com/m/56

Reply:
오 너무 좋은 기능이다..